### ISSUE TITLE
When the screen is in high contrast mode the keyboard focus is not visible on option present in the editor toolbar.

### ISSUE THREAD

**Issue by msftedad** [tinymce/tinymce#8907]
**What is the current behavior? Describe the bug**
When the screen is in high contrast mode the keyboard focus is not visible on option present in the editor toolbar (File,edit,--, redo, undo, headings.--).

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**
Pre-Requisite: Go to Settings-> Accessibility-> Contrast themes-> Toggle and Apply Aquatic and Desert mode.

1. Open URL https://codepen.io/Roopesh-T/pen/ZEmbPBK in Edge browser.
2. Page opens.
3. Navigate to Text Editor
4. Press alt+F10 using keyboard, focus will move to the tool bar present above editor.
5. Observe the issue that focus indicator is visible on editor toolbar (File,edit,--, redo, undo, headings.--) in High Contrast modes.

**What is the expected behavior?**
When the screen is in any high contrast mode(Aquatic, desert, dusk, night sky) the keyboard focus should be clearly visible on the items present in the editor toolbar (File,edit,--, redo, undo, headings.--).

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
Edge Version 114.0.1823.79 (Official build) (64-bit)
Window 11 22H2 (OS Build 22621.1848)

**Attachments:**

https://github.com/tinymce/tinymce/assets/93735775/9f0e576a-0317-4745-9d3e-9883ec183e67

**Comment by sumenaaz** [comment]
@Afraithe :What is the time frame your team is targeting to fix this issue? Could you please provide an estimated date?

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, As issue is still repro. Could you please provide update if there is any?
